Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -142,34 +146,34 @@ public static async Task<string> GetCurrentUserNameAsync(this UITestContext cont
         string password,
         string email,
         string tenant = "Default",
-        bool activateShell = true) => context.Application
-        .UsingScopeAsync(
-            async shellScope =>
-            {
-                var userService = shellScope.ServiceProvider.GetRequiredService<IUserService>();
-                var errors = new Dictionary<string, string>();
-                var user = await userService.CreateUserAsync(
-                    new User
-                    {
-                        UserName = userName,
-                        Email = email,
-                        EmailConfirmed = true,
-                        IsEnabled = true,
-                    },
-                    password,
-                    (key, error) => errors.Add(key, error));
-
-                if (user == null)
+        bool activateShell = true) =>
+            context.Application.UsingScopeAsync(
+                async serviceProvider =>
                 {
-                    var exceptionLines = new StringBuilder();
-                    exceptionLines.AppendLine("Create user error:");
-                    errors.ForEach(entry =>
-                        exceptionLines.AppendLine(CultureInfo.InvariantCulture, $"{entry.Key}: {entry.Value}"));
-                    throw new CreateUserFailedException(exceptionLines.ToString());
-                }
-            },
-            tenant,
-            activateShell);
+                    var userService = serviceProvider.GetRequiredService<IUserService>();
+                    var errors = new Dictionary<string, string>();
+                    var user = await userService.CreateUserAsync(
+                        new User
+                        {
+                            UserName = userName,
+                            Email = email,
+                            EmailConfirmed = true,
+                            IsEnabled = true,
+                        },
+                        password,
+                        (key, error) => errors.Add(key, error));
+
+                    if (user == null)
+                    {
+                        var exceptionLines = new StringBuilder();
+                        exceptionLines.AppendLine("Create user error:");
+                        errors.ForEach(entry =>
+                            exceptionLines.AppendLine(CultureInfo.InvariantCulture, $"{entry.Key}: {entry.Value}"));
```

Comment:
Since this is always a multiline exception it would be good it indent or bullet the subsequent lines, e.g.
```suggestion
                            exceptionLines.AppendLine(CultureInfo.InvariantCulture, $"- {entry.Key}: {entry.Value}"));
```